client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview/index.tsx

```diff
@@ -1,7 +1,99 @@
+import { Card, Badge } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import Layout from 'calypso/a8c-for-agencies/components/layout';
+import LayoutBody from 'calypso/a8c-for-agencies/components/layout/body';
+import LayoutHeader, {
+	LayoutHeaderTitle as Title,
+	LayoutHeaderSubtitle as Subtitle,
+	LayoutHeaderActions as Actions,
+} from 'calypso/a8c-for-agencies/components/layout/header';
+import LayoutTop from 'calypso/a8c-for-agencies/components/layout/top';
+import MobileSidebarNavigation from 'calypso/a8c-for-agencies/components/sidebar/mobile-sidebar-navigation';
+import getAgencyTierLogo from '../../lib/get-agency-tier-logo';
+import getTierBenefits from '../../lib/get-tier-benefits';
+
+import './style.scss';
+
 export default function AgencyTierOverview() {
+	const translate = useTranslate();
+
+	const title = translate( 'Your Agency Tier' );
+	const benefits = getTierBenefits( translate );
+
 	return (
-		<div>
-			<h1>Agency Tier Overview</h1>
-		</div>
+		<Layout
+			className="agency-tier-overview"
+			title={ title }
+			wide
+			sidebarNavigation={ <MobileSidebarNavigation /> }
+		>
+			<LayoutTop>
+				<LayoutHeader>
+					<Title>{ title } </Title>
+					<Subtitle>
+						{ translate( 'Experience the rewards of selling Auatomattic products and hosting.' ) }
+					</Subtitle>
+					<Actions>
+						{
+							// TODO: Add actions
+							<></>
+						}
+					</Actions>
+				</LayoutHeader>
+			</LayoutTop>
+
+			<LayoutBody>
+				<div className="agency-tier-overview__bottom-content">
+					<div className="agency-tier-overview__bottom-content-subheading">
+						{ translate( 'Take a closer look' ) }
+					</div>
+					<div className="agency-tier-overview__bottom-content-heading">
+						{ translate( 'Explore the benefits of the tiers' ) }
```

Comment:
From the screenshot, it looks like the section refers to a single tier. Should this be "Explore the benefits of the tier"? (Questions from the translation team)

Reply:
No, this is a section for all the tiers. The three icons on top indicate the tiers. We will have sections at the top that mention what tier each icon is for.